src/Logging/Logging.Console/src/ConsoleLogger.cs

```diff
@@ -137,11 +137,16 @@ private LogMessageEntry CreateDefaultLogMessage(StringBuilder logBuilder, LogLev
                 logBuilder.AppendLine(exception.ToString());
             }
 
+            string timestamp = null;
             var timestampFormat = Options.TimestampFormat;
+            if (timestampFormat != null)
+            {
+                timestamp = Options.UseUtcTimezone ? DateTime.UtcNow.ToString(timestampFormat) : DateTime.Now.ToString(timestampFormat);
```

Comment:
```suggestion
                var dateTime = Options.UseUtcTimezone ? DateTime.UtcNow : DateTime.Now;
                timestamp = dateTime.ToString(timestampFormat);
```
But I think this is more a matter of taste.

Reply:
Fine for me, will send update asap